modules/statics/src/networks.ts

```diff
@@ -226,6 +249,18 @@ class OfcTestnet extends Testnet implements OfcNetwork {
   explorerUrl = undefined;
 }
 
+class Rbtc extends Mainnet implements AccountNetwork {
```

Comment:
Is it appropriate to call it Rootstock and RootstockTestnet, respectively? This would be more aligned with the other network names.

Reply:
I rather not use specific names for the networks since they can change (tTRX recently did).
Even Tezos mainnet name changes per protocol. I'm not sure about this one, but I think it is safer to just use the coin name.